sdk/nestjs/src/VariableValue.ts

```diff
@@ -9,7 +8,7 @@ type VariableValueData = {
 
 export const VariableValue = createParamDecorator(
     (data: VariableValueData, ctx: ExecutionContext) => {
-        const req = getRequestFromContext(ctx)
+        const req = ctx.switchToHttp().getRequest()
 
         if (!req.dvc_client || !req.dvc_user) {
```

Comment:
should this be using CLS?

Reply:
or is it gonna have both CLS and the request properties?

Reply:
I was thinking it needed both since cls can't be injected here, but it seems like I can use ClsServiceManager to get the cls service